brand/schema.py

```diff
@@ -1,8 +1,10 @@
 import ast
+import hashlib
 import json
 import logging
 import re
 
+from django.core.cache import cache
```

Comment:
I think that you need to define CACHES in settings.py to make this work. Without it, I'm not sure what type of cache you're running. (In memory? DB? file-based?)

Here are the settings from the harvest repo. I would _not_ recommend copying them verbatim since the usecase there is quite specific. (In particular, the max entries and 30 day timeout are not what we're going for here.)

```
CACHES = {
    "default": {
        "BACKEND": "django.core.cache.backends.db.DatabaseCache",
        "LOCATION": "request_cache_table",
        "OPTIONS": {"MAX_ENTRIES": 15000, "CULL_FREQUENCY": 10},
        "TIMEOUT": 60 * 60 * 24 * 30,
    }
```

Reply:
only one server instance. I'd personally use a DB cache. It's slow but goign to be faster than hitting the DB each time. We might want to upgrade our server soon, in which case we'd then change to memcache